src/renderer/components/mulmo_viewer/tabs/slide_tab.vue

```
@@ -6,6 +6,15 @@
       <p class="text-muted-foreground mb-4 text-sm">{{ t("project.productTabs.slide.description") }}</p>
     </div>
     <div v-else>
+      <div class="flex w-full items-center justify-between">
+        <MediaPlayer
+          :videoWithAudioSource="lipSyncFiles?.[currentBeat?.id]"
+          :videoSource="movieFiles?.[currentBeat?.id]"
+          :imageSource="imageFiles?.[currentBeat?.id]"
+          :audioSource="audioFiles[currentLanguage]?.[currentBeat?.id]"
+          />
+        
```

ESLint

[error] Replace `··/>⏎········` with `/>`

(prettier/prettier)